recurly/response.go

```diff
@@ -16,8 +16,12 @@ type (
 		// Errors holds an array of validation errors if any occurred.
 		Errors []Error
 
+		// Transaction holds the transaction returned with a transaction error.
+		// This will only be populated when a TransactionError is present.
+		Transaction Transaction
```

Comment:
For the record: We decided to set `Transaction` and `TransactionError` on the `Response` object to pointers so that the caller could check if those fields were `nil`.